openquakeplatform/fabfile.py

```diff
@@ -425,6 +427,9 @@ def _set_auth():
     local('python manage.py loaddata '
           './openquakeplatform/common/fixtures/*.json')
 
+def _init_sql():
+    local('cat ./openquakeplatform/common/fixtures/*.sql | '
+          'python manage.py dbshell')
```

Comment:
This breaks CI and any automated installation since it requires user interaction:
```
[localhost] local: cat ./openquakeplatform/common/fixtures/*.sql | python manage.py dbshell
Password for user oqplatform: 
```